lib/tasks/data_preparation.rake

```diff
@@ -0,0 +1,16 @@
+namespace :data_preparation do
+  desc 'Recalculate post and topic score'
+  task calculate_score: :environment do
```

Comment:
cái này chạy khi nào e?

Reply:
cái này sẽ chạy khi anh seed xong db. vì mình chưa có callback để tính toán score cho vote.